js/views/chatHeadsVw.js

```diff
@@ -52,15 +70,25 @@ module.exports = baseVw.extend({
     }
 
     this.chatHeadViews = [];
+    this.renderChatHeads(this.nextChatToShow, this.showPerScroll);
+    this.$el.html(this.$headContainer);
 
-    this.collection.forEach((md, index) => {
+    return this;
+  },
+
+  renderChatHeads: function(start, end) {
+    var totalChats = this.collection.models.length,
```

Comment:
Rather than set this variable, can you just use `this.collection.length` (`models` not needed) directly on line 91?